types/lru-cache/index.d.ts

```diff
@@ -3,46 +3,47 @@
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
 //                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
 
 declare class LRUCache<K, V> implements Iterable<[K, V]> {
-    constructor(options?: LRUCache.Options<K, V>);
+    constructor(options: LRUCache.Options<K, V>);
 
     /**
      * Return total length of objects in cache taking into account `length` options function.
      *
-     * @deprecated use `cache.size` instead
-     * @since 7.0.0
+     * @deprecated since 7.0 use `cache.size` instead
```

Comment:
Removing the extra `@since` line because it doesn't mean `deprecated since`, it means `available since`